lib/dotiw/methods.rb

```diff
@@ -12,6 +12,7 @@ def distance_of_time_in_words_hash(from_time, to_time, options = {})
     end
 
     def distance_of_time(seconds, options = {})
+      options = options.dup
```

Comment:
This fixes only one place, we have other places where we do this, e.g. [right below](https://github.com/radar/distance_of_time_in_words/pull/119/files#diff-743e49728c7512377bde2db8ddfa47cc619bcf3b899809007e13ca7aabe99a8aR24).

Here's a potentially more elegant way of doing it:
 
```ruby
options = { include_sections: true }.merge(options_with_scope(options))
```

Reply:
Ah, I guess that Issue you made was for all the occurrences, not just the one. Thought I got away with an easy one there. ;)

> Here's a potentially more elegant way of doing it:
> 
> ```ruby
> options = { include_sections: true }.merge(options_with_scope(options))
> ```

We don't want to do that because we pass the "unscoped" options as the second param: 

```ruby
_display_time_in_words DOTIW::TimeHash.new(seconds, nil, nil, options_with_scope(options)).to_hash, options
```

So we need to retain options without calling `options_with_scope` there. 

I'll see what I can do to change all (most?) of the occurrences and how much of a time sink this might be...